packages/theme-chalk/src/switch.scss

```diff
@@ -4,64 +4,58 @@
 @use 'mixins/var' as *;
 @use 'common/var' as *;
 
-$switch-height: () !default;
-$switch-height: map.merge(
-  (
-    'large': 40px,
-    'default': 32px,
-    'small': 24px,
-  ),
-  $switch-height
+$switch-height: (
+  ) !default;
+$switch-height: map.merge(('large': 40px,
+      'default': 32px,
+      'small': 24px,
+    ),
+    $switch-height
```

Comment:
## Question

The logic is the same, but why change the code format? the same elsewhere

Reply:
My eslint automatically formatted it when I saved it. I will try to fix it